Explorer/Assets/Scripts/ECS/SceneLifeCycle/Systems/PartitionSceneEntitiesSystem.cs

```diff
@@ -140,7 +108,7 @@ private void PartitionNewEntity(in Entity entity, ref SceneDefinitionComponent d
             }
             else
             {
-                PartitionData partition = partitions[definition.InternalJobIndex];
+                var partition = partitionDataContainer.Partitions[definition.InternalJobIndex];
```

Comment:
Can we call `SetPartitionComponentData` instead? Seems to be this piece is duplicated

Reply:
Of course, changing